build.yaml

```diff
@@ -146,6 +152,33 @@ steps:
        to: /cluster-tests.tar.gz
    dependsOn:
     - hail_build_image
+ - kind: runImage
+   name: test_hail_java
+   image:
+     valueFrom: hail_run_image.image
+   script: |
+     set -ex
+     cd /io
+     mkdir -p src/test
+     tar xzvf resources.tar.gz -C src/test
+     export HAIL_TEST_SKIP_R=1
+     java -cp hail-test.jar org.testng.TestNG -listener is.hail.LogTestListener testng.xml
+     java -cp hail-test.jar org.testng.TestNG -listener is.hail.LogTestListener testng-cpp-codegen.xml
+   inputs:
+    - from: /resources.tar.gz
+      to: /io/resources.tar.gz
+    - from: /hail-test.jar
+      to: /io/hail-test.jar
+    - from: /testng.xml
+      to: /io/testng.xml
+    - from: /testng-cpp-codegen.xml
+      to: /io/testng-cpp-codegen.xml
+   outputs:
+    - from: /io/test-output
+      to: /test-output
```

Comment:
Does this go to the gas bucket by default? or is this on the local computer?

Reply:
These get copied from the local computer (in /io/test-output) into the CI service account bucket with the other artifacts for this build.